sample_usage/typecheck/fail/1_one_var_two_types.py

```diff
@@ -0,0 +1,2 @@
+x = 3
+x = 'abc'
```

Comment:
End all of these files with a blank line. (In general, you should configure your text editor to do this for you automatically.)